v2/nodejs/session/getjwtpayload.mdx

```diff
@@ -4,6 +4,12 @@ title: getJWTPayload
 hide_title: true
 ---
 
+:::caution
```

Comment:
You can strike out the title shown in the side bar since this is deprecated. See: https://github.com/supertokens/docs/pull/73/files#diff-3a5a56ae70d56703fb1554f5eab08bd911e8c7f6be8d99dca7ab5cc3cdf22668R3 as an example. (Google strike out text and use any online site that does that).

Reply:
Done